src/main/java/com/workat/api/review/dto/response/ReviewTypeListResponse.java

```diff
@@ -12,13 +14,13 @@
 public class ReviewTypeListResponse {
 
 	@ApiModelProperty(name = "response", notes = "리뷰 타입", example = "{\"response\":[{\"name\":\"PARKING\",\"content\":\"주차하기 편해요~\"},{\"name\":\"VIEW\",\"content\":\"뷰가 좋아요\"},{\"name\":\"NOT_CROWDED\",\"content\":\"한적해요~\"},{\"name\":\"NIGHT_VIEW\",\"content\":\"야경이 예뻐요~\"},{\"name\":\"SPACE\",\"content\":\"넓고 깨끗해요~\"},{\"name\":\"WIFI\",\"content\":\"와이파이가 빵빵해요!\"},{\"name\":\"POWER\",\"content\":\"콘센트 자리 많아요!\"},{\"name\":\"SEAT\",\"content\":\"좌석이 편해요\"},{\"name\":\"QUIET\",\"content\":\"조용한 공간이 있어요!\"},{\"name\":\"FOCUS\",\"content\":\"집중이 잘돼요\"}]}")
```

Comment:
```suggestion
	@ApiModelProperty(name = "response", notes = "리뷰 타입", example = "[{\"name\":\"PARKING\",\"content\":\"주차하기 편해요~\"},{\"name\":\"VIEW\",\"content\":\"뷰가 좋아요\"},{\"name\":\"NOT_CROWDED\",\"content\":\"한적해요~\"},{\"name\":\"NIGHT_VIEW\",\"content\":\"야경이 예뻐요~\"},{\"name\":\"SPACE\",\"content\":\"넓고 깨끗해요~\"},{\"name\":\"WIFI\",\"content\":\"와이파이가 빵빵해요!\"},{\"name\":\"POWER\",\"content\":\"콘센트 자리 많아요!\"},{\"name\":\"SEAT\",\"content\":\"좌석이 편해요\"},{\"name\":\"QUIET\",\"content\":\"조용한 공간이 있어요!\"},{\"name\":\"FOCUS\",\"content\":\"집중이 잘돼요\"}]")
```

요거 제가 잘못 작성한건데 요렇게 고쳐주실 수 있을까용??